Enh: add initial tutorial drafts (#115)
pyOpenSci/python-package-guide: tutorials/1-installable-code.md

# Make your Python code pip installable

> The first step in creating a Python package based on code that you

Author: ```suggestion
The first step in creating a R package based on code that we
```
